docs/datasets/index.rst

```diff
@@ -52,10 +52,106 @@ Note that in Gammapy, 2D image analyses are done with 3D cubes with a single
 energy bin, e.g. for modeling and fitting,
 see the `2D map analysis tutorial <./tutorials/image_analysis.html>`__.
 
-
-Getting Started
-===============
-
+To analyse multiple runs, you can either stack the datasets together, or perform
+a joint fit across multiple datasets.
+
+.. _stack:
+
+Stacking Multiple Datasets
+==========================
+
+Stacking datasets implies that the counts, background and reduced IRFs from all the
+runs are binned together to get one final dataset for which a likelihood is
+computed during the fit. Stacking is often useful to reduce the computation effort while
+analysing multiple runs.
+
```

Comment:
Maybe move the table here and just say "the following table lists how the individual quantities are handled..."

Reply:
I still think this might be the better place for he table: so focus on "here is how it works"  first and then later on "with this definition the npred computation is conserved". The latter is the non-trivial aspect...